### ISSUE TITLE
Incorrect Content Type IDs

### ISSUE THREAD

**Issue by andresbonilla** [contentful/contentful_express_tutorial#7]
In line 7 and 14 of services/product.js the content type ID is set to '2PqfXUJwE8qSYKuM0U6w8M' instead of 'product'. This matches the default test space but in the express app demo, but once you change the auth token and space ID to your new example space using the Product Catalogue example space template, the app fails. The error also doesn't help at all. The Product Catalogue sample space uses 'product' for the content type ID, not '2PqfXUJwE8qSYKuM0U6w8M'. The express app demo requires this value to be 'product' but doesn't mention that the sample code has the wrong content type IDs. Same thing happens for categories.

**Comment by jordan314** [comment]
+1 thanks for this. Following the tutorial, if I change my app accessToken and space, it just hangs with a generic error in the console:
```
(node:5701) UnhandledPromiseRejectionWarning: #<Object>
(node:5701) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). (rejection id: 1)
(node:5701) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
```

**Comment by cachrisman** [comment]
this is fixed with #14.
